src/tests/ftest/pool/list_verbose.py

```diff
@@ -110,7 +132,7 @@ def create_expected(self, pool, scm_free, nvme_free, scm_imbalance,
                     "imbalance": nvme_imbalance
                 },
             ],
-            "mem_file_bytes": scm_size
+            "mem_file_bytes": 0 if self.is_pmem_mode() else scm_size
```

Comment:
We have a built-in way to determine if the server is configured with MD on SSD:
```suggestion
            "mem_file_bytes": scm_size if self.server_managers[0].manager.job.using_control_metadata else 0
```

Reply:
+1

Reply:
great thanks for pointing that out, I was looking but didn't manage to find it